Makefile.am

```diff
@@ -79,6 +79,7 @@ noinst_HEADERS = \
 
 EXTRA_libffi_la_SOURCES = \
 	src/aarch64/ffi.c src/aarch64/sysv.S				\
+	src/aarch64/ffi.c src/aarch64/win64_armasm.S	\
```

Comment:
`src/aarch64/ffi.c` is already listed here on the line above, so it doesn't need to be repeated.